extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -632,7 +662,7 @@ public function canEdit() {
 	 * @return bool
 	 */
 	public function isAuthor( $user ) {
-		if ( $this->mUser ) {
+		if ( $this->loadRevisionsAndAuthor( true ) && $this->mUser ) {
```

Comment:
I forgot to pass `$master` arg to `loadRevisionsAndAuthor` method - fixed in 10c816a.